versioned_docs/version-2.0/how_to_guides/datasets/manage_datasets_in_application.mdx

```diff
@@ -72,3 +72,22 @@ To do so, select a dataset, click on "Examples", and then click the "Export Data
 This will open a modal where you can select the format you want to export to.
 
 ![Export Dataset Modal](../static/export-dataset-modal.png)
+
+## Create and manage dataset splits
+
+Dataset splits are divisions of your dataset which you can use to segment your data. For example, it is common
+in machine learning workflows to split datasets into test, training, and validation sets. This can be useful
+to prevent overfitting - where a model performs well on the training data but poorly on unseen data. In evaluation
+workflows, it can be useful to do this when you have a dataset with multiple categories that you may want to evaluate
+separately; or if you are testing a new use case that you may want to include in your dataset in the future, but want
+to keep separate for now.
```

Comment:
For a user, when would I want to include something in a new split vs. as metadata?

LIke let's say I have data that all has 'language' attribution. Should I put that in separate splits? Or should I add as metadata within a single split?